### DISCUSSION TITLE
Support Commodore 128 (Commodore 1571 drive) ?

### DISCUSSION THREAD

**Discussion by gamed1** [keirf/greaseweazle#220] [Q&A]
Hello, I got some floppies which I want to read.
What I know:
- Single Sided
- Double Density
- 5 1/4 inch
- written on it: "128 RBS datadisk"

Assumptions:
- It are floppies with data from Superbase: https://en.wikipedia.org/wiki/Superbase_(database)
- It is superbase 128 for the commodore 128

What I have tried:
- tested my system with a test floppy (IBM), this works dine.
- used all supported formats of greaseweazle   -> no one can it.

My question:
- Assuming it is from the Commodore 128, how can I read these floppies?

**Comment by kyousif87** [comment]
I'm wondering about that too...
How can I hookup a 1571 to grease weasel so I can archive C128 floppy disks?

**Comment by keirf** [comment]
You don't need a 1571 drive, and PC 5.25 drive should do. But if the disks are GCR coded then Greaseweazle can't analyse that currently. You can dump to Kryoflux raw format and analyse using Kryoflux DTC tool which can produce G64 files I believe. Or dump to SCP and use DDR64 https://csdb.dk/release/?id=218378

**Reply by kyousif87** [reply]
Unfortunately, I do not have an IBM PC compatible 5.25" FDD.
I have tons of PC 5.25" disks but no drive for them.

**Reply by keirf** [reply]
As with the 1541 there's basically no chance of interfacing a 1571 to anything but dedicated controller hardware.

**Comment by d235j** [comment]
If you have access to macOS you can also use the Applesauce software to analyze and export D64 or G64; both KF raw and SCP will work as input. Alternatively, the Fluxengine software can output D64.

Note that flippy disks will likely need a modded drive as described here: http://cowlark.com/fluxengine/doc/Index_sensor_mod_FDD_1.1.pdf

**Reply by keirf** [reply]
Oh yes good point. Lots of things can analyse basic C64 GCR actually. Really I should get it added to GW. I did check with David Given that I can crib from fluxengine (and reimplement in Python, obviously) if necessary.
